static/app/gettingStartedDocs/python/python.tsx

```diff
@@ -274,14 +277,69 @@ sentry_sdk.init(
   nextSteps: () => [],
 };
 
+const profilingOnboarding: OnboardingConfig = {
+  install: (params: Params) => [
+    {
+      type: StepType.INSTALL,
+      description: tct('Install our Python SDK using [code:pip]:', {
+        code: <code />,
+      }),
+      configurations: [
+        {
+          description:
+            params.docsLocation === DocsPageLocation.PROFILING_PAGE
+              ? tct(
+                  'You need a minimum version [code:1.18.0] of the [code:sentry-python] SDK for the profiling feature.',
+                  {
+                    code: <code />,
+                  }
+                )
+              : undefined,
+          language: 'bash',
+          code: getInstallSnippet(),
+        },
+      ],
+    },
+  ],
+  configure: (params: Params) => [
+    {
+      type: StepType.CONFIGURE,
+      description: t(
+        "Import and initialize the Sentry SDK early in your application's setup:"
+      ),
+      configurations: [
+        {
+          language: 'python',
+          code: getSdkSetupSnippet(params),
+        },
+      ],
+    },
+  ],
+  verify: () => [
+    {
+      type: StepType.VERIFY,
+      description: t(
+        'Verify that profiling works by running the profiler around functions of different duration.'
+      ),
+      configurations: [
+        {
+          language: 'python',
+          code: getProfilingVerifySnippet(),
```

Comment:
It looks like we lost the note about continuous profiling, is that on purpose?
![CleanShot 2024-11-27 at 13 10 11](https://github.com/user-attachments/assets/5d896d86-bcb6-493f-a351-8bfbbfbd257b)

Reply:
cc. @Zylphrex